source/manual/business-readiness-finder.html.md

```diff
@@ -0,0 +1,61 @@
+
+For a list of all of the YAML files that drive the finder and its related pages, see: [Publish changes to the business readiness finder](/manual/business-readiness-publish-changes.html).
+
+## Overview
+
+The full journey for the business readiness finder is made up of the following content items:
+
+1. The start page: [`/business-uk-leaving-eu`](https://www.gov.uk/business-uk-leaving-eu)
+2. The Q&A: [`/prepare-business-uk-leaving-eu`](https://www.gov.uk/prepare-business-uk-leaving-eu)
+3. The finder: [`/find-eu-exit-guidance-business`](https://www.gov.uk/find-eu-exit-guidance-business)
+4. The email signup page: [`/find-eu-exit-guidance-business/email-signup`](https://www.gov.uk/find-eu-exit-guidance-business/email-signup)
+
+Editors can request changes to the finder via a [form](https://support.integration.publishing.service.gov.uk/eu_exit_business_readiness_request/new) in the [Support](https://github.com/alphagov/support) app.
+
+### Start page
+
+A normal page which can be updated in Mainstream Publisher.
+
+### Q&A
+
+The Q&A is [documented in finder-frontend](https://github.com/alphagov/finder-frontend/blob/master/docs/q-and-a.md).
+
+### Finder
+
+#### Facets
+
+The finder uses the search-api to find and filter results. Whilst standard finders get more granular as you select values in different facets (the number of results goes down), the business finder gets more _broad_ (number of results goes up).
+
+Standard finders retrieve content published by a single publishing app, where the ability to add facets is built directly into the publishing interface. The result is facets are stored under `details.facets` in the [finder's content item](https://www.gov.uk/api/content/cma-cases). Think of `details` as containing the body of the content.
+
+The business readiness finder collates content from _multiple_ publishing apps, not all of which have facets built into their interface. However, content-tagger provides a central way of tagging to links (which themselves come from the Publishing API), so the business finder uses content tagger to avoid having to add facets to the interface of every relevant publishing app. The [business finder content item](https://www.gov.uk/api/content/find-eu-exit-guidance-business) therefore has facet information under `links.facet_group`. Think of `links` as containing metadata describing the content.
+
+Link-based facets are filtered by 'OR logic' by default, as opposed to the 'AND logic' exhibited in other finders.
+
+#### Grouped display
+
+The business finder uses links for facets, where there is no concept of a link hierarchy or grouping. In order to group links into separate facets, such as "Sector / Business Area", [groupings are defined in YAML](https://github.com/alphagov/content-tagger/blob/master/lib/data/find-eu-exit-guidance-business.yml) in the content tagger. Any changes to this YAML file affect both the facets in the finder and the options in the Q&A.
```

Comment:
I'm not sure what this paragraph is trying to say.

Reply:
What I'm trying to say is all facet links are thrown into the same pot. There's no association between, say, `Aerospace` and its parent `Sector / Business area`. So we have to manually group the facet values into their facets using the YAML file. The concept isn't there in the facet representation itself; `Aerospace` and say `Employ EU citizens` may or may not be related to each other, we have no way of knowing without looking at the YAML file. 

Unless I've misunderstood how it works?

Reply:
That makes more sense. Use that instead.

Reply:
But again, this is something that should be moved to finder-frontend.